botocore/docs/client.py

```diff
@@ -143,9 +143,17 @@ def _add_client_methods(self, client_methods):
 
     def _add_client_method(self, section, method_name, method):
         section.add_title_section(method_name)
-        method_section = section.add_new_section(method_name)
+        full_method_name = f'{self._client_class_name}.Client.{method_name}'
+        method_section = section.add_new_section(
+            method_name,
+            context={'full_method_name': full_method_name},
```

Comment:
I'm wondering if we pass something more flexible like "qualifier" that can be joined with the method name as needed? I'm not sure how much that expands the PR, just a thought to avoid the redundant name handling.

Reply:
Added the suggested changes in a new commit.